Tests/test_file_avif.py

```diff
@@ -146,7 +146,7 @@ def test_write_rgb(self, tmp_path: Path) -> None:
 
             # avifdec hopper.avif avif/hopper_avif_write.png
             assert_image_similar_tofile(
-                reloaded, "Tests/images/avif/hopper_avif_write.png", 6.02
+                reloaded, "Tests/images/avif/hopper_avif_write.png", 6.14
```

Comment:
Why do we need to relax the average pixel difference threshold?

Reply:
This change is no longer present. See https://github.com/python-pillow/Pillow/pull/8858#discussion_r2029662183